src/app/views/sap/register.php

```diff
@@ -9,14 +9,7 @@
     <link href="https://fonts.googleapis.com/css?family=Crete+Round" rel="stylesheet" type="text/css">
 </head>
 <body>
-    <div class="navbar collapsed" style="position:static;">
-        <a class="navbar-title" href="javascript:void(0)">Felicity ʼ16</a>
-        <ul class="pull-right">
-            <li><a href="<?= base_url() ?>">Home</a></li>
-        </ul>
-        <a href="javascript:void(0)" class="menu-toggle-button">&equiv;</a>
-        </a>
-    </div>
+    <?php $this->load_fragment('sap/navbar_fragment', ['logged_in' => $logged_in]); ?>
```

Comment:
It already passes whatever is passed to the view. So, I don't think you need to pass `logged_in` again.